### PR TITLE
Update for 0.991.0 release

### PR DESCRIPTION
**Pull request by aashikam**
## Purpose
Fixes sources to comply with latest syntax

### CODE REVIEW

File: guide/phone_store/phone_store_service.bal [hunk 1]
```diff
@@ -61,171 +52,177 @@ import ballerina/log;
...
+        var payload = request.getJsonPayload();
+        // Try parsing the JSON payload from the request.
+        if (payload is json) {
+            // Valid JSON payload.
+            requestPayload = payload;
+        } else {
+            // NOT a valid JSON payload.
+            response.statusCode = 400;
+            response.setJsonPayload({ "Message": "Invalid payload - Not a valid JSON payload" });
+            checkpanic caller->respond(response);
```
Comment (riyafa) [inline guide/phone_store/phone_store_service.bal]: Any reason for having removed the `done` here?

File: guide/phone_store/phone_store_service.bal [hunk 2]
```diff
@@ -61,171 +52,177 @@ import ballerina/log;
...
+        json contact = requestPayload.ContactNumber;
+        json phoneName = requestPayload.PhoneName;
+
+        // If payload parsing fails, send a "Bad Request" message as the response.
         if (name == null || address == null || contact == null || phoneName == null) {
             response.statusCode = 400;
             response.setJsonPayload({ "Message": "Bad Request - Invalid payload" });
-            _ = caller->respond(response);
-            done;
+            checkpanic caller->respond(response);
```
Comment (riyafa) [inline guide/phone_store/phone_store_service.bal]: I think previously `done` was there for a reason. Can we verify please?

**Comment by riyafa** [comment]
Shall we update the travis.yml also for the 0.991.0 version?

**Comment by aashikam** [comment]
travis.yml updated for the 0.991.0 version

**Comment by aashikam** [comment]
travis.yml updated for 0.991.0 version

File: guide/phone_store/phone_store_service.bal (hunk 1 above)
Comment (aashikam) [reply guide/phone_store/phone_store_service.bal]: I added `return`

File: guide/phone_store/phone_store_service.bal (hunk 2 above)
Comment (aashikam) [reply guide/phone_store/phone_store_service.bal]: Added `return` clause

**Pull request merged**